onebusaway-android/src/main/java/org/onebusaway/android/util/UIUtils.java

```diff
@@ -1432,11 +1432,12 @@ public static void closeKeyboard(Context context, View v) {
      * for a stop.  See #700.
      *
      * @param response response from arrivals-and-departures-for-stop API
+     * @param filter   list of routes to not retrieve service alerts for
```

Comment:
I believe the docs here are inverted from the actual behavior - it should read `list of route_ids to retrieve service alerts for, or null to retrieve service alerts for all routes.  Note that this filter only affects alerts scoped to routes - it does not affect alerts scoped to stops or agencies`.  

The route filter here and in ArrivalsListFragment (and related adapters and util classes) contains the list of routes that you *should* show to the rider, not the ones you're filtering out.  Looking at the code again that's not clearly documented elsewhere - feel free to update those docs too if you'd like (but not required for this PR).